src/grammar.coffee

```diff
@@ -268,6 +271,7 @@ grammar =
   Param: [
     o 'ParamVar',                               -> new Param $1
     o 'ParamVar ...',                           -> new Param $1, null, on
+    o '... ParamVar',                           -> new Param $2, null, on    
```

Comment:
I think that allowing both ways is a pretty bad idea. It will lead to inconsistent code, the need for custom linters to tell people what to stick to and not bikeshed. Either we should switch to ES6 style or keep CS style, but not allow both. I am inclined to a switch, as it will make it easier for people to go between JS and CS, and it should be easy to codemod (I assume that CoffeeScript 2 has more subtle backwards incompatibility problems, if that's not the case, we should just stick to CS style).